homeassistant/const.py

```diff
@@ -181,6 +181,7 @@
 CONF_SOURCE: Final = "source"
 CONF_SSL: Final = "ssl"
 CONF_STATE: Final = "state"
+CONF_STATE_CLASS: Final = "state_class"
```

Comment:
Is state_class for all types of entities ? my impression is that it is only for sensors, and thus should be defined in sensors/__init__.py

Reply:
moved it there.